### PR TITLE
WIP: correct Data Cleaning code

### PR DESCRIPTION
**Pull request by jdpye**
Working through the examples and correcting code snippets now. Fixed some of the R datetime examples, work in progress until I get the Python ones tested as well.

### CODE REVIEW

**Comment by jdpye** [comment]
Because I'm not a good branch juggler today, it seems I'll be sneaking a draft of the acoustics lesson in as well.

**Comment by MathewBiddle** [comment]
Who do we contact at https://www.marinespecies.org/aphia.php?p=match to check the service?

**Comment by MathewBiddle** [comment]
something went haywire with the formatting to jekyll
image
debugging now.

**Comment by jdpye** [comment]
Ah sorry about that, I'm flying blind editor-wise, didn't local-build before i pushed.

**Comment by MathewBiddle** [comment]
No worries, that's why I'm here. I'll fix the syntax stuff 😄

File: _episodes/07-special-data-type-acoustic-telemetry.md
Comment (MathewBiddle) [inline _episodes/07-special-data-type-acoustic-telemetry.md:1]: This should be moved to the `_extras/` folder to sit next to eDNA.

File: _episodes/08-validation-and-publishing.md
Comment (MathewBiddle) [inline _episodes/08-validation-and-publishing.md:1]: this should be reverted back to `07`

File: _episodes/09-post-workshop.md
Comment (MathewBiddle) [inline _episodes/09-post-workshop.md:1]: this should be reverted back to `08`

**Review by MathewBiddle — APPROVED** [review]
I made some changes. Mainly to add telemetry as an extra instead of an episode. Limits the amount of file name changing we have to do.

**Review by MathewBiddle — APPROVED** [review]
merging

**Pull request merged**